books/ion-1-1/src/binary/values/timestamp.md

```diff
@@ -0,0 +1,446 @@
+## Timestamps
+
+Timestamps have two encodings:
+
+1. [**Short-form timestamps**](#short-form-timestamps), a compact representation optimized for the most commonly used precisions and date ranges.
+2. [**Long-form timestamps**](#long-form-timestamps), a less compact representation capable of representing any timestamp in the Ion data model.
+
+`0xEB x04` represents `null.timestamp`.
+
+##### Encoding of `null.timestamp`
+```
+┌──── Opcode 0xEB indicates a typed null; a byte follows specifying the type
+│  ┌─── Null type: timestamp
+│  │
+EB 04
+```
+
+> [!NOTE]
+> In Ion 1.0, text timestamp fields were encoded using the local time while binary timestamp fields were encoded
+> using UTC time. This required applications to perform conversion logic when transcribing from one format to the other.
+> *In Ion 1.1, all binary timestamp fields are encoded in local time.*
+
+
+### Short-form Timestamp
+
+If an opcode has a high nibble of `0x8_`, it represents a short-form timestamp. This encoding focuses on making the
+most common timestamp precisions and ranges the most compact; less common precisions can still be expressed via
+the variable-length [long form timestamp](#long-form-timestamp) encoding.
+
+Timestamps may be encoded using the short form if they meet all of the following conditions:
+
+The year is between 1970 and 2097.:: The year subfield is encoded as the number of years since 1970. 7 bits are
+dedicated to representing the biased year, allowing timestamps through the year 2097 to be encoded in this form.
+The local offset is either UTC, unknown, or falls between `-14:00` to `+14:00` and is divisible by 15 minutes.
+7 bits are dedicated to representing the local offset as the number of quarter hours from -56 (that is: offset `-14:00`).
+The value `0b1111111` indicates an unknown offset. At the time of this writing (2024-08T),
+[all real-world offsets fall between `-12:00` and `+14:00` and are multiples of 15 minutes](https://en.wikipedia.org/wiki/List_of_UTC_offsets).
+The fractional seconds are a common precision. The timestamp's fractional second precision (if present) is
+either 3 digits (milliseconds), 6 digits (microseconds), or 9 digits (nanoseconds).
+
+#### Opcodes by precision and offset
+
+Each opcode with a high nibble of `0x8_` indicates a different precision and offset encoding pair.
+
+| Opcode | Precision        | Serialized size in bytes\* | Offset encoding                             |
```

Comment:
Style suggestion—`mdbook` has built-in support for [footnotes](https://rust-lang.github.io/mdBook/format/markdown.html#footnotes), which could be used here instead of `\*`.